cosmo_tester/test_suites/cluster/full_cluster_test.py

```diff
@@ -221,6 +222,56 @@ def test_workflow_resume_manager_failover(minimal_cluster,
     assert installs[0]['status'] == 'completed'
 
 
+def test_replace_certificates_on_cluster(full_cluster, logger, ssh_key,
+                                         test_config, module_tmpdir):
+    broker1, broker2, broker3, db1, db2, db3, mgr1, mgr2 = full_cluster
+
+    example = get_example_deployment(mgr1, ssh_key, logger,
+                                     'cluster_replace_certs', test_config)
+    example.inputs['server_ip'] = mgr1.ip_address
+    example.upload_and_verify_install()
+    _validate_cluster_and_agents(mgr1, example.tenant)
+
+    for host in broker1, broker2, broker3, db1, db2, db3, mgr1, mgr2:
+        key_path = join('~', '.cloudify-test-ca',
+                        host.private_ip_address + '.key')
+        mgr1.run_command('cfy_manager generate-test-cert'
+                         ' -s {0},{1}'.format(host.private_ip_address,
+                                              host.ip_address))
+        mgr1.run_command('chmod 444 {0}'.format(key_path), use_sudo=True)
+    replace_certs_config_path = '~/certificates_replacement_config.yaml'
+    _create_replace_certs_config_file(mgr1, replace_certs_config_path,
+                                      ssh_key.private_key_path)
+
+    local_new_ca_path = join(str(module_tmpdir), 'new_ca.crt')
+    mgr1.get_remote_file('~/.cloudify-test-ca/ca.crt', local_new_ca_path)
+    mgr1.client._client.cert = local_new_ca_path
```

Comment:
We only update the client "locally" during the replace_certificates procedure (as seen [here](https://github.com/cloudify-cosmo/cloudify-cli/blob/5e2365dedce24daf3ea5bed445a4f450efe7d75b/cloudify_cli/replace_certificates_config.py#L178)), so we need to update it "again".